pkg/memlimit/memlimit_test.go

```diff
@@ -0,0 +1,31 @@
+package memlimit
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetMemoryLimit_ROX_MEMLIMIT(t *testing.T) {
+	// 4Gi
+	total := 4 * 1024 * 1024 * 1024
```

Comment:
nit: we have a const for this :)
https://github.com/stackrox/stackrox/blob/980f992efa68467123ac8d6ab65789e5feaf95a4/pkg/size/size.go#L7